docker/README.md

```diff
@@ -51,13 +106,16 @@ Or, visit it in your browser by opening http://localhost:8080
 
 ## Configuration
 
-Configuration is expected to be mounted to the exact path of `/opt/trino/gateway-ha-config.yml`.
+Configuration is expected to be mounted to the exact path
+of `/opt/trino/gateway-ha-config.yml`.
 If it is not mounted then the gateway will fail to initialize.
 
 ## Health Checking
 
-By default the container health checking is done by the [/usr/lib/trino/bin/health-check](./bin/health-check)
-script which simply expects a 2XX response from the server at `/api/public/backends`.
+By default the container health checking is done by
+the [/usr/lib/trino/bin/health-check](./bin/health-check)
+script which simply expects a 2XX response from the server
+at `/api/public/backends`.
 
 ## Building a custom Docker image
```

Comment:
todo: can delete this section as it's been reworded+moved to be the first

Reply:
Done